aws/provider.go

```diff
@@ -892,6 +893,7 @@ func Provider() *schema.Provider {
 			"aws_securityhub_member":                                  resourceAwsSecurityHubMember(),
 			"aws_securityhub_product_subscription":                    resourceAwsSecurityHubProductSubscription(),
 			"aws_securityhub_standards_subscription":                  resourceAwsSecurityHubStandardsSubscription(),
+			"aws_serverlessrepository_stack":                          resourceAwsServerlessRepositoryStack(),
```

Comment:
I apologize if we've discussed this before, but these SAR resources and data sources should probably be expanded to the full service naming per the Contribution Guide, unless there is a blocking reason not to (length be darned 🙁 ).

```go
"aws_serverlessapplicationrepository_stack": resourceAwsServerlessApplicationRepositoryStack(),
```

We may also want to consider adding "CloudFormation" Stack in here to match the API CreateCloudFormationStack naming, in case SAR releases some other form of "stacks" (such as its own).

```go
"aws_serverlessapplicationrepository_cloudformation_stack": resourceAwsServerlessApplicationRepositoryCloudFormationStack(),
```